thoth/glyph/lib.py

```diff
@@ -0,0 +1,113 @@
+import logging
+import os
+from os import path
+from thoth.common import init_logging
+from fasttext import load_model
+
+import pandas as pd
+from pygit2 import Repository
+from pygit2 import GIT_SORT_TOPOLOGICAL
+
+import time
+import datetime
+import sys
+
+init_logging()
+_LOGGER = logging.getLogger("glyph")
```

Comment:
```python
_LOGGER = logging.getLogger(__name__)
```

Reply:
`from thoth.glyph import __title__
`
doesn't seem to be working. 

__title__ is defined in __init__.py:
`__title__ = "glyph"`

Reply:
it's `__name__`

Reply:
Hmm..I was trying to use __title__ as it was defined in __init__.py
__name__ works though